PowerEditor/src/Notepad_plus.cpp

```diff
@@ -6195,6 +6195,7 @@ void Notepad_plus::getCurrentOpenedFiles(Session & session, bool includUntitledD
 			const wchar_t* langName = languageName.c_str();
 			sessionFileInfo sfi(buf->getFullPathName(), langName, buf->getEncoding(), buf->getUserReadOnly(), buf->getPosition(editView), buf->getBackupFileName().c_str(), buf->getLastModifiedTimestamp(), buf->getMapPosition());
 
+			sfi._hasClone = MainFileManager.bufferHasClone(buf);

```

Comment:
@molsonkiko,

How about `sfi._hasClone = (k == 1 && _mainDocTab.getIndexByBuffer(buf) != -1)`?

(I haven't tested this line).

Reply:
Good idea.
While we're here, one thing that troubles me about this code block is the fact that the variable `size_t k` is shadowed a few lines earlier in the block `if (languageName.empty())`. This is a case of code smell at the very least, and at worst a potential bug.

I think this problem should be solved (by renaming the looping variable in the `if (languageName.empty())` block) whether or not my PR is accepted, but especially if I'm going to make use of `size_t k` here.

Reply:
@molsonkiko,

Isn't `k` within the `if (languageName.empty())` block a completely different variable in a different scope?
But your point is good from a readability point of view, and I suppose Don would agree to replace that `k`.